source/mod.ts

```diff
@@ -0,0 +1,5 @@
+export * from "./types.ts";
+
+export * from "./html.ts";
+export * from "./markdown.ts";
+export * from "./markdownv2.ts";
```

Comment:
Explicitness = easier navigation.
```suggestion
export type { Formatter } from "./types.ts";

export { html } from "./html.ts";
export { markdown } from "./markdown.ts";
export { markdownv2 } from "./markdownv2.ts";
```

Reply:
true. On the other side it's nice to just export in the file instead of here too. 🤔

But I think this is independent from this PQ of providing Deno support.